src/Core/Content/Cms/DataResolver/CmsSlotsDataResolver.php

```diff
@@ -34,24 +36,43 @@ class CmsSlotsDataResolver
      */
     private $definitionRegistry;
 
+    /**
+     * @var EventDispatcherInterface
+     */
+    private $eventDispatcher;
+
```

Comment:
Use typed properties

Reply:
If you are already working on it, it would be nice if you change all the other properties too.

Reply:
@Dominik28111 ok sure.

Reply:
@Dominik28111 done